java/ql/src/Security/CWE/CWE-501/TrustBoundaryFixed.java

```diff
@@ -0,0 +1,8 @@
+public void doGet(HttpServletRequest request, HttpServletResponse response) {
+    String username = request.getParameter("username");
+
+    if (validator.isValidInput("HTTP parameter", username, "username", 20, false)) {
+        // GOOD: The input is sanitized before being written to the response.
```

Comment:
`session` instead of `response`?

```suggestion
        // GOOD: The input is sanitized before being written to the session.
```